rM/oecore-raw.nix

```diff
@@ -1,20 +1,34 @@
-{ stdenv, fetchurl, libarchive, python, which, file }:
+{ stdenv, fetchurl, libarchive, python, file, which }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "rm-toolchain-bin";
   version = "1.8-23.9.2019";
-  src = import <nix/fetchurl.nix> {
-    url = "https://remarkable.engineering/oecore-x86_64-cortexa9hf-neon-toolchain-zero-gravitas-1.8-23.9.2019.sh";
-    sha256 = "cbd5ffa0e2ee4b244d5780289d3950f6228d8e3372f90c0dd728b45201ca61e6";
+
+  src = fetchurl {
+    url = "https://remarkable.engineering/oecore-x86_64-cortexa9hf-neon-toolchain-zero-gravitas-${version}.sh";
```

Comment:
I think you should mark this package x86_64 only.

Reply:
Thanks!